Doc/extending/newtypes.rst

```diff
@@ -965,14 +957,15 @@ Finalization and De-allocation
 
 This function is called when the reference count of the instance of your type is
 reduced to zero and the Python interpreter wants to reclaim it.  If your type
-has memory to free or other clean-up to perform, put it here.  The object itself
-needs to be freed here as well.  Here is an example of this function::
+has memory to free or other clean-up to perform, you can put it here.  The
```

Comment:
Added "you can".
https://docs.python.org/3/extending/newtypes.html#finalization-and-de-allocation